### PR TITLE
Add new troubleshooting issue for 715

### PR DESCRIPTION
**Pull request by micgibso**
Signed-off-by: micgibso <micgibso@uk.ibm.com>

### CODE REVIEW

**Comment by micgibso** [comment]
@jagraj @makandre @sghung Review please.

**Comment by makandre** [comment]
I don't think the issue is specific to Appsody, Windows, or Eclipse

I think the underlying issue of Codewind not detecting the port can happen in other situations too, but perhaps the fact that Appsody projects typically spends more timing in the `starting` phase, Appsody projects are more likely to expose this problem.

File: docs/_documentations/troubleshooting.md [hunk 1]
```diff
@@ -365,6 +365,30 @@ These steps reproduce the issue:
...
+Codewind version: 0.5.0
+OS: Windows
+IDE extension version: 0.5.0
+IDE version: Eclipse 2019-09
+Issue link: https://github.com/eclipse/codewind/issues/715
+-->
+## Appsody projects stuck in starting state
+If you work on Appsody projects on Windows using the Eclipse IDE, you might see Appsody projects stuck in the `Starting` or `Stopped` state even though the container logs say the projects are up and running. This can happen when you create a number of projects using the default and Appsody templates with Codewind 0.5.0. 
+
+These steps reproduce the issue:
```
Comment (makandre) [inline docs/_documentations/troubleshooting.md]: Are the reproduction steps needed as part of the troubleshooting?  I think we should re-word this to let users know in what situations are they likely to run into this problem, e.g. after reinstalling Codewind and they open up to their workspace that has, many projects, etc.

Comment (jagraj) [reply docs/_documentations/troubleshooting.md]: Makes sense, we do not need to list reproduction steps.

Comment (micgibso) [reply docs/_documentations/troubleshooting.md]: Ok, have removed the reproduction steps.

**Comment by micgibso** [comment]
@jagraj @makandre @sghung Have removed Appsody in particular, made the whole thing more generic, and moved it from the Appsody section into the creating projects section within the Troubleshooting. Please review.

File: docs/_documentations/troubleshooting.md [hunk 2]
```diff
@@ -113,6 +112,23 @@ The Codewind Che extension might lose connectivity to the Codewind pod during a
 
 **Workaround:** Refresh the projects list to have the tree repopulate. If the issue persists, refresh the webpage.
 
+<!--
+Action/Topic: Appsody with Codewind
```
Comment (makandre) [inline docs/_documentations/troubleshooting.md]: Should Appsody here also be removed?

Comment (micgibso) [reply docs/_documentations/troubleshooting.md]: Yes, on it...

Comment (micgibso) [reply docs/_documentations/troubleshooting.md]: Updated, have replaced that string with the string for the section

**Comment by micgibso** [comment]
Have referenced eclipse/codewind-docs#188 in the new issue https://github.com/eclipse/codewind/issues/812.

**Pull request merged**